include/settings.hpp

```diff
@@ -234,10 +275,10 @@ class basic_settings : public settings_base {
         if constexpr (refl::is_reflectable<Node>()) {
             std::lock_guard lg(_lock);
             for (const auto &[localKey, localValue] : parameters) {
-                const auto &key    = localKey;
-                const auto &value  = localValue;
-                bool        is_set = false;
-                for_each(refl::reflect(*_node).members, [&, this](auto member) {
+                const auto &key                 = localKey;
+                const auto &value               = localValue;
+                bool        is_set              = false;
+                auto        iterate_over_member = [&, this](auto member) {
```

Comment:
Not a fan of the naming `iterate_over_member` - the names should reflect what the function does - reading `for_each(..., iterate_over_member)` doesn't say much

Reply:
me neither... just refactored this out because this lambda needs to be called twice: once for the base class and once for the user block implementation.

Reply:
@ivan-cukic feel free to make a proposal how and/or change these names directly.